specification/batch/data-plane/Microsoft.Batch/stable/2018-03-01.6.1/BatchService.json

```diff
@@ -11447,7 +11447,7 @@
         }
       },
       "title": "The definition of the user identity under which the task is run.",
-      "description": "Specify either the userName or autoUser property, but not both."
+      "description": "Specify either the userName or autoUser property, but not both. The Windows user logon type will be Interactive mode for Azure PaaS VMs, and be Batch mode for Azure IaaS VMs."
```

Comment:
"and be": replace by "and will be" or "and"

Reply:
We don't use PaaS/IaaS in public documentation.

Can you say this instead?
`On CloudServiceConfiguration pools, this user is logged in with the INTERACTIVE flag. On Windows VirtualMachineConfiguration pools, this user is logged in with the BATCH flag.`

Reply:
We DO use PaaS/IaaS in the document, which exactly for cloudServiceConfiguration/virtualMachineConfig.